### PR TITLE
Integration Test for getComponentGroups method

### PR DESCRIPTION
**Pull request by farmaazon**
[ci no changelog needed]

### Pull Request Description

This PR contains minimal integration with new engine's method and an integration test printing the method's return value. It was written as a part of https://www.pivotaltracker.com/story/show/181743571 

### Important Notes

The test requires 2022.1.1-nightly.2022-04-26 engine version or later.

### Checklist

Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [x] If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run.sh ide dist` and `./run.sh ide watch`.

### CODE REVIEW

File: integration-test/tests/engine.rs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use enso_gui::initializer::setup_global_executor;
+use enso_web::sleep;
+use std::time::Duration;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+struct TestOnNewProjectControllersOnly {
```
Comment (wdanilo) [inline integration-test/tests/engine.rs]: no sections

File: integration-test/tests/engine.rs [hunk 2]
```diff
@@ -0,0 +1,58 @@
+use enso_gui::integration_test::prelude::*;
```
Comment (wdanilo) [inline integration-test/tests/engine.rs:3]: no docs

**Review by wdanilo — APPROVED** [review]

File: app/gui/controller/engine-protocol/src/language_server.rs [hunk 3]
```diff
@@ -129,12 +129,12 @@ trait API {
...
     /// Move the execution context up the stack.
-    #[MethodInput=PopFromExecutionContextInput,rpc_name="executionContext/pop"]
-    fn pop_from_execution_context(&self, context_id:ContextId) -> ();
+    #[MethodInput=PopFromExecutionContextInput, rpc_name="executionContext/pop"]
+    fn pop_from_execution_context(&self, context_id: ContextId) -> ();
 
     /// Attach a visualisation, potentially preprocessed by some arbitrary Enso code, to a given
     /// node in the program.
-    #[MethodInput=AttachVisualisationInput,rpc_name="executionContext/attachVisualisation"]
+    #[MethodInput=AttachVisualisationInput, rpc_name="executionContext/attachVisualisation"]
```
Comment (akavel) [inline app/gui/controller/engine-protocol/src/language_server.rs:137]: [nit] why some of the lines now have a space before `rpc_name=`, while others don't?

File: integration-test/tests/engine.rs [hunk 4]
```diff
@@ -0,0 +1,70 @@
+/// The test suite of IDE-engine communication. The view is not instantiated, and controllers
```
Comment (akavel) [inline integration-test/tests/engine.rs]: Shouldn't this be `//!` ?

File: integration-test/tests/engine.rs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            defined_on_type: main_module,
+            name:            MAIN_DEFINITION_NAME.to_owned(),
+        },
+        this_argument_expression:         None,
+        positional_arguments_expressions: vec![],
+    });
+    ls_json_connection.push_to_execution_context(&execution_ctx.context_id, &frame).await.unwrap();
+    sleep(Duration::from_secs(15)).await;
+    let groups = ls_json_connection.get_component_groups(&execution_ctx.context_id).await.unwrap();
+    DEBUG!("{groups:?}");
```
Comment (akavel) [inline integration-test/tests/engine.rs:69]: (Question to better understand: what does this line give us here? Is it intended to be left here?)

Comment (farmaazon) [reply integration-test/tests/engine.rs:69]: It is logged in console, so we can check how the response from the Engine look like. I would keep it there.

**Review by akavel — APPROVED** [review]

**Pull request merged**